### ISSUE TITLE
Make Nextcloud a full operable PWA

### ISSUE THREAD

**Issue by David100mark** [nextcloud/server#13879]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by MorrisJobke** [comment]
Duplicate of #8563

**Comment by David100mark** [comment]
i don't think that it is an duplicate... #8563 is only referencing to Chrome-PWAs, while this isn't the web-standard. On top of that, #8563 is closed...

**Comment by 0cwa** [comment]
This is not a duplicate issue. PWA support means the webpage would be cached locally and load instantly, #8563 was only to make the top bar collapse.  Please reopen this, Nextcloud could be so much snappier on desktop....
